mbedtls-sys/src/types.rs

```diff
@@ -84,18 +84,19 @@ pub mod raw_types {
     }
 }
 
-#[cfg(feature = "libc")]
+#[cfg(all(feature = "libc", not(target_env = "sgx")))]
 extern crate libc;
-#[cfg(feature = "libc")]
+#[cfg(all(feature = "libc", not(target_env = "sgx")))]
 mod libc_types {
     pub use super::libc::FILE;
 }
 
-#[cfg(not(feature = "libc"))]
+#[cfg(not(any(feature = "libc", target_env = "sgx")))]
 mod libc_types {
     pub enum FILE {}
 }
 
+#[cfg(not(target_env = "sgx"))]
```

Comment:
This should is always be needed

Reply:
Reverted back changes in this file. Will push after resolving other comments.